scripts/chart-templates/crd-assume-ownership/values.yaml

```diff
@@ -7,5 +7,5 @@ global:
     systemDefaultRegistry: ""
 
 image:
-  repository: rancher/kubectl
-  tag: v1.18.6
+  repository: rancher/rancher-agent
+  tag: v2.4-head
```

Comment:
Update to use `-latest`, `-head` could potentially be broken at any point since it is not an official release.